packages/paste-core/components/tooltip/__test__/index.spec.tsx

```diff
@@ -53,7 +53,40 @@ describe('Tooltip', () => {
       if (tooltip === null) {
         return;
       }
-      expect(tooltip.getAttribute('hidden')).toBeDefined();
+      expect(tooltip.getAttribute('hidden')).not.toBeNull();
```

Comment:
swapping to not be null because if it is null, to be defined is still true, always.